ebook/en/content/030-github-marksdown-cheatsheet.md

```diff
@@ -0,0 +1,243 @@
+GitHub Markdown Cheatsheet
+===================
+
+- - - - 
+# Heading 1 #
```

Comment:
I believe that the way the tags are defined here, it will break the eBook semantics, can you maybe restructure this and wrap the examples in code snippets?